pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -1590,8 +1590,8 @@ let
         mktplcRef = {
           name = "java";
           publisher = "redhat";
-          version = "1.3.0";
-          sha256 = "sha256-Y5hP/Rq9BsFwbCRQWOfiLHKoYkKBpZx8blg9o74obfk=";
+          version = "1.4.0";
+          sha256 = "sha256-9q3ilNukx3sQ6Fr1LhuQdjHHS251SDoHxC33w+qrfAI=";
```

Comment:
```suggestion
          sha256 = "0fzdkwjs79marnj2rzw207s5isgibgw8jrkdqs666w99q65nzjij";
```

Reply:
I've updated all hashes to the `sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=` format.

Reply:
That is what ofborg gave me...

Reply:
The update script still uses the old format, which output I copy pasted.

Reply:
The format does not matter. Somehow the hash behind it which can have multiple formats, was wrong.